lib/components/button-group.js

```diff
@@ -0,0 +1,41 @@
+import { mergeData } from "../utils";
+import { arrayIncludes } from "../utils/array";
+
+export const props = {
+    vertical: {
+        type: Boolean,
+        default: false
+    },
+    size: {
+        type: String,
+        default: null,
+        validator: size => arrayIncludes(["sm", "", "lg"], size)
+    },
+    tag: {
+        type: String,
+        default: "div"
+    },
+    ariaRole: {
+        type: String,
+        default: "group"
+    }
+};
+
+export default {
+    functional: true,
+    props,
+    render(h, { props, data, children }) {
+        return h(
+            props.tag,
+            mergeData(data, {
+                staticClass: "btn-group",
+                class: {
+                    "btn-group-vertical": props.vertical,
+                    [`btn-group-${props.size}`]: Boolean(props.size)
+                },
+                attrs: { "aria-role": props.ariaRole }
```

Comment:
Gotcha. I copied that from something else, so I'll see if I can track down the other incorrect case.